### DISCUSSION TITLE
What should I put in .env "your token"?

### DISCUSSION THREAD

**Discussion by TheFreeRider** [hmes98318/r6-discord-bot#8] [General]
Im a beginner of programming, want to try and make a discord bot for my friends.

What should I write inside the .env

TOKEN = "your token"

the "your token"?

**Comment by hmes98318** [comment]
It might be helpful to check out the following tutorial video.  
https://youtu.be/Q7Hgp6bg0kI  

Create your bot: https://discord.com/developers/applications/
